ci_framework/roles/openshift_login/tasks/main.yml

```diff
@@ -49,8 +49,48 @@
     cifmw_openshift_login_api: "{{ cifmw_openshift_login_api_out.stdout }}"
     cifmw_openshift_login_context: "{{ cifmw_openshift_login_context_out.stdout }}"
     cifmw_openshift_login_user: "{{ cifmw_openshift_login_user_out.stdout }}"
+    cifmw_openshift_kubeconfig: "{{ cifmw_openshift_login_kubeconfig }}"
     cifmw_openshift_api: "{{ cifmw_openshift_login_api_out.stdout }}"
     cifmw_openshift_context: "{{ cifmw_openshift_login_context_out.stdout }}"
     cifmw_openshift_user: "{{ cifmw_openshift_login_user_out.stdout }}"
     cifmw_openshift_token: "{{ cifmw_openshift_login_token | default(omit) }}"
+    cifmw_install_yamls_environment: >-
+      {{  cifmw_install_yamls_environment |
+          combine({'KUBECONFIG': cifmw_openshift_login_kubeconfig})
+      }}
     cacheable: true
+
+- name: Create the openshift_login parameters file
+  tags:
+    - always
+  vars:
+    cifmw_openshift_login_params_content: >-
+      cifmw_openshift_kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+      cifmw_openshift_api: "{{ cifmw_openshift_api }}"
+      cifmw_openshift_context: "{{ cifmw_openshift_context }}"
+      cifmw_openshift_user: "{{ cifmw_openshift_user }}"
+      {% if cifmw_openshift_token is defined %}
```

Comment:
wondering if you shouldn't use `{%- if ... %}` in order to trim the space.
Same for the `{% endif %}`

Reply:
Seems to be working, my fault was the `>` instead of the proper ` |` to flag yaml identation.